src/components/PostTable/PostTable.js

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import config, { timezoneDifference } from '../../config';
+import Header from './Header';
+import {
+  Wrapper, Title, Row, Cell, Link,
+} from './PostTable.styles';
+
+const { linkPrefix } = config.postTable;
+
+function PostTable({ activeCell, data }) {
+  const { day, hour } = activeCell;
+
+  const timeToMiliseconds = 1000;
+  const timeToUserTimezone = (utcTime) => (
+    new Date((utcTime + timezoneDifference) * timeToMiliseconds).getMinutes()
+  );
+  const compareCreationTime = (a, b) => (
+    timeToUserTimezone(a.created_utc) - timeToUserTimezone(b.created_utc)
+  );
+
+  const posts = (day > -1 && day !== null)
+    ? [...data[day][hour]].sort(compareCreationTime)
+    : false;
+
+  const Table = () => (
+    posts.map((post) => (
+      <Row key={post.id}>
+        <Cell cellName="title">
+          <Link
+            href={`${linkPrefix}${post.permalink}`}
+            target="_blank"
+            rel="noopener noreferrer"
+          >
+            {post.title}
+          </Link>
+        </Cell>
+        <Cell cellName="time">
+          {
+            new Date((post.created_utc + timezoneDifference) * 1000)
+              .toLocaleString('en-US', {
+                hour: 'numeric', minute: 'numeric', hour12: true,
+              }).toLowerCase()
+          }
+        </Cell>
+        <Cell cellName="score">
+          {post.num_comments}
+        </Cell>
+        <Cell cellName="comments">
+          {post.score}
+        </Cell>
+        <Cell cellName="author">
+          {
+            post.author
```

Comment:
Ah cool, you do this check already. But I guess the author is always set. If it's deleted `author === '[deleted]'`